coordinator/core/core.go

```diff
@@ -352,3 +492,76 @@ func (c *Core) generateSecrets(ctx context.Context, secrets map[string]Secret) (
 
 	return newSecrets, nil
 }
+
+func (c *Core) generateCertificateForSecret(secret Secret, key interface{}) (x509.Certificate, error) {
+	// Load given information from manifest as template
+	var template *x509.Certificate
+	var err error
+
+	template = &secret.Cert
+
+	// Define or overwrite some values for sane standards
+	if template.DNSNames == nil {
+		template.DNSNames = c.cert.DNSNames
+	}
+	if template.IPAddresses == nil {
+		template.IPAddresses = c.cert.IPAddresses
```

Comment:
```suggestion
		template.IPAddresses = util.DefaultCertificateIPAddresses
```
and add `var DefaultCertificateIPAddresses = []net.IP{net.IPv4(127, 0, 0, 1), net.IPv6loopback}` to util/tls.go and also replace all other occurrences